src/ansys/templates/python/pyace_pckg/README.rst

```diff
@@ -0,0 +1,200 @@
+Py{{ cookiecutter.product_name }} {{ cookiecutter.library_name }}
```

Comment:
This file will not be included anywhere since it does not live inside a `{{ cookiecutter. ... }}/` directory. Also, it looks like it is referring to the PyAnsys Product Library layout.